pytorch_lightning/trainer/trainer.py

```diff
@@ -1433,6 +1455,21 @@ def __set_ckpt_path(self, ckpt_path: Optional[str], model_provided: bool, model_
             # load best weights
             ckpt_path = self.checkpoint_callback.best_model_path
 
+        if ckpt_path == "last":
+            candidates = [ft.ckpt_path for ft in ft_checkpoints] + [
+                cb.last_model_path for cb in self.checkpoint_callbacks
+            ]
+            candidates_fs = {path: get_filesystem(path) for path in candidates if path}
+            candidates_ts = {path: fs.modified(path) for path, fs in candidates_fs.items() if fs.exists(path)}
```

Comment:
I believe we should only take the last_model_path from the main checkpoint callback (the first one).

Reply:
We were talking with @carmocca about this and the conclusion was that at this moment, it does not really matter as they all point to the same file and in the future, once issue #5030 is done, then it will actually make sense the load the latest available from multiple runs. Does it make sense to you as well?

Reply:
Yes. Also note that when multiple checkpoint instances are used, not all might have `save_last=True` set